product_secondary_unit/views/product_views.xml

```diff
@@ -10,7 +10,7 @@
         <field name="groups_id" eval="[(4, ref('product.group_uom'))]"/>
         <field name="arch" type="xml">
             <xpath expr="//page[@name='general_information']" position="inside">
-                <group string="secondary unit of measure">
+                <group string="secondary unit of measure" groups="product_secondary_unit.group_secondary_unit">
```

Comment:
This shouldn't be done here, but on `groups_id` in the view (where you have right now the `product.group_uom` one).